### DISCUSSION TITLE
ERROR in fetching `gem.api`  " Nokogiri"

### DISCUSSION THREAD

**Discussion by Suvraneel** [lowlighter/metrics#990] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/63473496/163691031-70a5a763-81f7-4772-a6f5-c6a887ef21e1.png)

The workflows suddenly stopped working... Can't figure out why
https://github.com/Suvraneel/Suvraneel/runs/6049788014?check_suite_focus=true

**Comment by lowlighter** [comment]
This issue happened in the past, it was related to the nokogiri not being compatible with the ruby version present on the container.
Not sure if it's the same here, where they released a new version which dropped support for the ruby interpreter installed

**Comment by lowlighter** [comment, marked as answer]
I noticed you use `Suvraneel/metrics@latest` (which 738 commits behind this repo)

If you use a fork, you need to use `Suvraneel/metrics@master` instead (which is the default branch).

You can also rebase your `latest` branch to be up-to-date with upstream branch, though when you have a fork you already control the versionning anyway so it's to directly use `master` branch instead

**Reply by Suvraneel** [reply]
@lowlighter   Yep didn't notice that.
Yes its working perfectly. Apologies for bothering.

Thanks 🖤

**Reply by lowlighter** [reply]
No worries, thanks a lot for using metrics 👍
